### PR TITLE
Add project 'Remix' button for creating copies of projects

### PR DESCRIPTION
**Pull request by joshlory**
This change adds a 'Remix' button to standalone projects, as well as legacy share links:

Going forward users won't be able to create new `/c/123` share links (LevelSources table), only `/p/artist#VIMYQWp28AvMpVK4oP7n6g` links (channels API).

### CODE REVIEW

File: dashboard/app/assets/javascripts/header.js.erb [hunk 1]
```diff
@@ -259,6 +277,8 @@ dashboard.showProjectHeader = function () {
     });
   });
 
+  $('.project_remix').click(remixProject);
```
Comment (laurelfan) [inline dashboard/app/assets/javascripts/header.js.erb]: how does the .project_remix button get added to the dom for the non-minimal project header?

Comment (joshlory) [inline dashboard/app/assets/javascripts/header.js.erb]: Good catch, I removed this by accident.

File: shared/js/initApp.js [hunk 2]
```diff
@@ -291,6 +297,11 @@ function initApp() {
       appOptions.hideSource = true;
       appOptions.callouts = [];
     }
+  } else if (appOptions.isLegacyShare) {
```
Comment (laurelfan) [inline shared/js/initApp.js]: doesn't need to be fixed now, but I think this conditional is getting a bit on the too-branchy-to-read side

**Comment by laurelfan** [comment]
looks good! :+1:

**Pull request merged**